database/query/profile_status.sql

```diff
@@ -137,21 +138,23 @@ SELECT
     rt.id AS rule_type_id,
     rt.guidance as rule_type_guidance,
     rt.display_name as rule_type_display_name
-FROM rule_evaluations res
-         LEFT JOIN eval_details ed ON ed.rule_eval_id = res.id
-         LEFT JOIN remediation_details rd ON rd.rule_eval_id = res.id
-         LEFT JOIN alert_details ad ON ad.rule_eval_id = res.id
-         INNER JOIN repositories repo ON repo.id = res.repository_id
-         INNER JOIN rule_type rt ON rt.id = res.rule_type_id
-WHERE res.profile_id = $1 AND
+FROM latest_evaluation_statuses les
+         INNER JOIN evaluation_rule_entities ere ON ere.id = les.rule_entity_id
+         INNER JOIN eval_details ed ON ed.id = les.evaluation_history_id
+         INNER JOIN remediation_details rd ON rd.evaluation_id = les.evaluation_history_id
+         INNER JOIN alert_details ad ON ad.evaluation_id = les.evaluation_history_id
+         INNER JOIN rule_instances AS ri ON ri.id = ere.rule_id
+         INNER JOIN rule_type rt ON rt.id = ri.rule_type_id
+         LEFT JOIN repositories repo ON repo.id = ere.repository_id
```

Comment:
I'm not thrilled about this pile of joins, but I don't have a substantially better solution at the moment.